c2rust-analyze/src/main.rs

```diff
@@ -636,6 +653,23 @@ fn run(tcx: TyCtxt) {
         info.acx_data.set(acx.into_data());
     }
 
+    let (shim_call_rewrites, shim_fn_def_ids) = rewrite::gen_shim_call_rewrites(&gacx, &gasn);
```

Comment:
Can this never panic?

Reply:
Added a comment:
```Rust
    // This call never panics, which is important because this is the fallback if the more
    // sophisticated analysis and rewriting above did panic.
```